client/tos.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import TosModal from 'tos-modal';
```

Comment:
Nitpick: Missing `Internal dependencies` comment.

Reply:
👍 Added in https://github.com/Automattic/woocommerce-payments/pull/983/commits/4afaeb405a56f0fbe53660ddc0eb2f27b4df7664.